src/azure-cli/azure/cli/command_modules/appservice/_validators.py

```diff
@@ -336,7 +336,7 @@ def validate_vnet_integration(cmd, namespace):
                                                      resource_group_name=namespace.resource_group_name)
 
         sku_name = plan_info.sku.name
-        disallowed_skus = {'FREE', 'SHARED', 'BASIC', 'ElasticPremium', 'PremiumContainer', 'Isolated', 'IsolatedV2'}
+        disallowed_skus = {'FREE', 'SHARED', 'PremiumContainer', 'Isolated', 'IsolatedV2'}
```

Comment:
Can we confirm with function apps team this list accurate?

Reply:
We confirmed this on the original email thread with the functions team that prompted this change